zapisy/apps/users/admin.py

```diff
@@ -37,6 +39,31 @@ def get_queryset(self, request) -> QuerySet:
         qs = super(StudentAdmin, self).get_queryset(request)
         return qs.select_related('program', 'user')
 
+    actions = ['refresh_opening_times']
+
+    def refresh_opening_times(self, request, queryset):
+        """Refreshes opening times for selected students."""
+        semester = Semester.get_upcoming_semester(True)
```

Comment:
Czytelniejsze będzie
```suggestion
        semester = Semester.get_upcoming_semester(strict=True)
```